src/test/kotlin/io/github/mojira/arisa/modules/KeepPrivateModuleTest.kt

```diff
@@ -13,14 +14,24 @@ import io.kotest.matchers.shouldBe
 private val REMOVE_SECURITY = mockChangeLogItem(
     created = RIGHT_NOW.minusSeconds(10),
     field = "security",
-    changedFromString = "10318",
```

Comment:
Tests were failing because they are using `"private"` as private security level of the mocked issue while these changelog items used `"10318"` (so tests were flawed previously as well); have now added a constant `PRIVATE_SECURITY_LEVEL` holding that security level.

Reply:
Ahhhh, thanks for finding this. I completely missed it myself.